src/Template/Layout/default.ctp

```diff
@@ -6,13 +6,12 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
     <!-- Base CSS -->
-    <?= $this->Html->css('CrudView.bootstrap.min');?>
-    <?= $this->Html->css('CrudView.bootflat.min');?>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css">
```

Comment:
> I'd rather not include a gigantic js/css file in our repo - especially minified.

I am not suggesting that.

Acceptable solutions from my perspective are:

 * allow configuring css/js files - including external js/css files in config
 * installing/downloading that dependency in a build step, referencing it locally

In that order.